backend/controllers/groupDelete.js

```diff
@@ -1,23 +1,33 @@
 const groupDeleteFactory = (getGroupDetailsApi, deleteGroupApi, maintainUrl) => {
-  const stashStateAndRedirectToIndex = (req, res, errors, group) => {
+  const stashStateAndRedirectToIndex = (req, res, errors, group, url) => {
```

Comment:
can't see this extra parameter used anywhere?

Reply:
It is correct.  The call to stashStateAndRedirectToIndex needed an update (if attempt to manipulate URL to delete a group with child groups e.g. /manage-groups/{Group with Children}/delete/children/none).  Fixed.